src/main/kotlin/mods/eln/mechanical/RotaryMotor.kt

```diff
@@ -24,6 +24,7 @@ import net.minecraft.nbt.NBTTagCompound
 import org.lwjgl.opengl.GL11
 import java.io.DataInputStream
 import java.io.DataOutputStream
+import java.util.*
```

Comment:
Fully qualify

Reply:
... I didn't even tell IntelliJ to do any auto corrections in those files, fixed the warnings manually, how on Earth???  Might want to change the formatter options for the project?